jboss-eap-config-jgroups/module.yaml

```diff
@@ -31,8 +31,8 @@ envs:
       example: "0.0.0.0"
       description: "Override the default HA bind address (default value is the result of 'hostname -i') with a specific address."
     - name: JGROUPS_PING_PROTOCOL
-      example: "openshift.DNS_PING"
-      description: "Protocol to discover other nodes in this cluster. Example values: openshift.DNS_PING, openshift.KUBE_PING, kubernetes.KUBE_PING, dns.DNS_PING"
+      example: "dns.DNS_PING"
```

Comment:
This isn't a default, right? It's just an example.

The change is fine.

Reply:
Right yeah, the var here doesn't have a default. The default was openshift.KUBE_PING, now kubernetes.KUBE_PING in the ha.sh script.

Not sure why the description got dropped here though, will double check.

Reply:
Never mind, the description is still there, its below this part of the diff...